.github/workflows/ci.yml

```diff
@@ -687,6 +690,9 @@ jobs:
       uses: ./.github/actions/install_runner
     - name: Pull mitmproxy image
       run: docker pull datadog/system-tests:proxy-v0
```

Comment:
Oops, my `rg proxy-v0` ignored dotdirs. Fixed.

Reply:
As expected, CI now fails because `datadog/system-tests:proxy-v1` is nonexistent, thus the above pull fails.

Reply:
I have worked around that with https://github.com/DataDog/system-tests/pull/1109/commits/cf81be6440a0bcc83a841cf4f46749ab939c6c0e

Reply:
Note that Circle CI still fails because it doesn't handle the GH tag for rebuilding the proxy image:

```
INTERNALERROR> requests.exceptions.HTTPError: 404 Client Error: Not Found for url: http+docker://localhost/v1.41/images/datadog/system-tests:proxy-v1/json
```

Reply:
Could you  add a step that always rebuild the proxy in circleCI (it's ok here, the job is only here to ensure that system-tests is working on circleci) ?

Reply:
Fixed in 3e68a4aa